ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -138,7 +138,8 @@ private[state] object Conversions {
       submitter = Party.assertFromString(subInfo.getSubmitter),
       applicationId = LedgerString.assertFromString(subInfo.getApplicationId),
       commandId = LedgerString.assertFromString(subInfo.getCommandId),
-      maxRecordTime = parseTimestamp(subInfo.getMaximumRecordTime)
+      maxRecordTime = parseTimestamp(subInfo.getMaximumRecordTime),
+      deduplicateUntil = parseTimestamp(subInfo.getDeduplicateUntil).toInstant,
```

Comment:
Can we pick a default here if it is missing? e.g. make this backwards compatible.

It really doesn't matter so much with this change at this point in time, but I want us to get into habit to correctly do these changes. We'll soon be at a point when we'll enable data continuity testing, which will of course help to force the issue.